docs/citing.rst

```diff
@@ -1,46 +1,31 @@
-Citations in the pysat ecosystem
-================================
-
-When referring to this software package, please cite the original paper by Stoneback et al [2018] https://doi.org/10.1029/2018JA025297 as well as the package https://doi.org/10.5281/zenodo.3475498. Note that this doi will always point to the latest version of the code.  A list of dois for all versions can be found at the [zenodo page](https://zenodo.org/record/3475499).
-
-Example for citation in BibTex for a generalized version:
-
-.. code::
-
-  @misc{pysatMissions,
-    author       = {Stoneback, R.A. and
-                    Klenzing, J.H. and
-                    Burrell, A.G. and
-                    Depew, M. and
-                    Spence, C.},
-    title        = {Pysat Mission Planning Toolkit (pysatMissions) vX.Y.Z},
-    month        = oct,
-    year         = 2019,
-    doi          = {10.5281/zenodo.3475499},
-    url          = {https://doi.org/10.5281/zenodo.3475499}
+Citation Guidelines
+===================
+
+When publishing work that uses pysatMissions, please cite the package and
+any package it depends on that plays an important role in your analysis.
+Specifying which version of pysatMissions used will also improve the
+reproducibility of your presented results.
+
+pysatMissions
+-------------
+
+* Jeff Klenzing, Russell Stoneback, Angeline Burrell, Matthew Depew,
+  & Carey Spence. (2020, July 29). pysat/pysatMissions: Version 0.2.1
+  (Version v0.2.1). Zenodo. http://doi.org/10.5281/zenodo.3966508
+
+.. code-block:: latex
+
+  @software{pysatMissions,
+  author       = {Jeff Klenzing and
+                  Russell Stoneback and
+                  Angeline Burrell and
+                  Matthew Depew and
+                  Carey Spence},
```

Comment:
Recommend Surname, Other Names and ... format.

Reply:
To aid maintainability, we could just read the author list from the zenodo.json file.  Should we use the surname first format there as well?

Reply:
Hmmmm.... maybe?

Reply:
Sorry.  Responding to the wrong comment.  Fixing this here